model_analyzer/config/input/config_command.py

```diff
@@ -217,17 +218,38 @@ def _check_no_search_values(self, args: Namespace,
                 '\nPlease use brute search mode or remove batch size search values.'
             )
 
-    def _check_no_list_values(self, args: Namespace,
-                              yaml_config: Optional[Dict[str, List]]) -> None:
+    def _check_no_global_list_values(
+            self, args: Namespace, yaml_config: Optional[Dict[str,
+                                                              List]]) -> None:
         concurrency = self._get_config_value('concurrency', args, yaml_config)
         batch_sizes = self._get_config_value('batch_sizes', args, yaml_config)
 
         if concurrency or batch_sizes:
             raise TritonModelAnalyzerException(
                 f'\nProfiling of models in quick search mode is not supported with lists of concurrencies or batch sizes.'
-                '\nPlease use brute search mode or remove concurrency/batch_sizes list.'
+                '\nPlease use brute search mode or remove concurrency/batch sizes list.'
             )
 
+    def _check_no_per_model_list_values(
+            self, args: Namespace, yaml_config: Optional[Dict[str,
+                                                              List]]) -> None:
+        profile_models = self._get_config_value('profile_models', args,
+                                                yaml_config)
+
+        if type(profile_models) is str:
+            return
+
+        for model in profile_models.values():
+            if not 'parameters' in model:
```

Comment:
parameters is for PA args. You have that covered. What about model_config_parameters?

```
profile_models:
  model_1:
    model_config_parameters:
      max_batch_size: 2
```

The existence of ANY model_config_parameters probably breaks us for quick search.